reactive/containerd.py

```diff
@@ -257,21 +300,75 @@ def publish_config():
     )
 
 
-@when_not('containerd.br_netfilter.enabled')
-def enable_br_netfilter_module():
+@when('endpoint.docker-registry.ready')
+@when_not('containerd.registry.configured')
+def configure_registry():
     """
-    Enable br_netfilter to work
-    around https://github.com/kubernetes/kubernetes/issues/21613
+    Add docker registry config when present.
 
-    :returns: None
+    :return: None
     """
-    try:
-        modprobe('br_netfilter', persist=True)
-    except Exception:
-        log(traceback.format_exc())
-        if host.is_container():
-            log('LXD detected, ignoring failure to load br_netfilter')
-        else:
-            log('LXD not detected, will retry loading br_netfilter')
-            return
-    set_state('containerd.br_netfilter.enabled')
+    registry = endpoint_from_flag('endpoint.docker-registry.ready')
+
+    # Handle TLS data
+    cert_subdir = registry.registry_netloc
+    if registry.has_tls():
+        # Ensure the CA that signed our registry cert is trusted.
+        install_ca_cert(registry.tls_ca, name='juju-docker-registry')
+
+    manage_registry_certs(cert_subdir, remove=False)
```

Comment:
same comment as before -- i think you'll need a full path here vs the netloc subdir.